python/ray/experimental/state/api.py

```diff
@@ -146,6 +146,208 @@ def list(
 
         return response["data"]["result"]
 
+    def get(
+        self,
+        resource: StateResource,
+        id: str,
+        options: Optional[GetApiOptions],
+        _explain: bool = False,
+    ) -> Optional[
+        Union[
+            ActorState,
+            PlacementGroupState,
+            NodeState,
+            WorkerState,
+            TaskState,
+            ObjectState,
```

Comment:
It should be Union[ObectState, List[ObjectState]]?

Reply:
I see - was thinking of returning a list even if there is only a single `ObjectState` entry. But `Union` will work.